drools-drl/drools-drl-parser-tests/src/test/java/org/drools/drl/parser/antlr4/MiscDRLParserTest.java

```diff
@@ -3654,4 +3654,37 @@ void nullSafeDereferencingMethodCallBindVariable() {
         ExprConstraintDescr constraintDescr = (ExprConstraintDescr) patternDescr.getConstraint().getDescrs().get(0);
         assertThat(constraintDescr.toString()).isEqualToIgnoringWhitespace("$containsL : address!.city.contains(\"L\")");
     }
+
```

Comment:
```suggestion

    @Test
```
Missing `@Test` annotation.